widgets/bat_openbsd.lua

```diff
@@ -0,0 +1,74 @@
+-- {{{ Grab environment
+local setmetatable = setmetatable
+local tonumber = tonumber
+local string = { format = string.format, gmatch = string.gmatch }
+local io = { popen = io.popen }
+local math = { floor = math.floor }
+local helpers = require("vicious.helpers")
+local string = {
+    gmatch = string.gmatch,
+    match = string.match,
+    format = string.format
+
+}
```

Comment:
IMHO it'd look better if you switch line 12 and 13.

Reply:
Definitely! I don't know how I couldn't see that line.